test/utils-test.js

```diff
@@ -202,7 +202,7 @@ describe('BN.js/Utils', function () {
       assert.equal(new BN(0x3fffffe).cmpn(0x3fffffd), 1);
       assert.throws(function () {
         new BN(0x3fffffe).cmpn(0x4000000);
```

Comment:
It works because there's `cmpn` after `new`, so there's side-effect-only constructor, which `semistandard` reports about.